src/pretalx/cfp/views/wizard.py

```diff
@@ -127,6 +129,19 @@ def get_context_data(self, **kwargs):
         step_list.append({'phase': 'todo', 'label': _('Done!'), 'icon': 'check'})
         context['step_list'] = step_list
 
+        if step == 'info':
+            # use value of query string parameters track and submission_type as initial values for form if they are valid
+            for field, model_name in (('submission_type', SubmissionType), ('track', Track)):
+                request_value = self.request.GET.get(field)
+                try:
+                    request_id = model_name.id_from_slug(request_value) if request_value else None
+                except:
+                    continue
+                # search requested object by ID
+                obj = model_name.objects.filter(event=self.request.event, id=request_id).first()
+                # ensure that the whole slug matches, not only the ID
+                if obj and obj.slug() == request_value:
```

Comment:
But we can't be sure that this is true: The slug is calculated off `str(obj.name)` after all, which can differ between active locales. I'd just not validate this here.